skimage/restoration/tests/test_inpaint.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function, division
+
+import numpy as np
+from numpy.testing import (run_module_suite, assert_allclose,
+                           assert_raises)
+from skimage.restoration import inpaint
+
+
+def test_inpaint_biharmonic():
+    img = np.tile(np.square(np.linspace(0, 1, 5)), (5, 1))
+    mask = np.zeros_like(img)
+    mask[2, 2:] = 1
+    mask[1, 3:] = 1
+    mask[0, 4:] = 1
+    out = inpaint.inpaint_biharmonic(img, mask)
+    ref = np.array(
+        [[0., 0.0625, 0.25000000, 0.5625000, 0.73925058],
+         [0., 0.0625, 0.25000000, 0.5478048, 0.76557821],
+         [0., 0.0625, 0.25842878, 0.5623079, 0.85927796],
+         [0., 0.0625, 0.25000000, 0.5625000, 1.00000000],
+         [0., 0.0625, 0.25000000, 0.5625000, 1.00000000]]
+    )
+    assert_allclose(ref, out)
```

Comment:
You could probably do a similar simple 3D example, to check that nD is working? You just need to stack a second array, no need to make a cube. =)